packaging/nnfw.spec

```diff
@@ -150,6 +150,13 @@ NPU daemon for optimal management of NPU hardware
 %define option_test -DENABLE_TEST=ON -DENVVAR_ONERT_CONFIG=ON
 %endif # test_build
 
+%if %{build_type} == Debug
+%if %{npud_build} == 1
+# ENVVAR_NPUD_CONFIG: Use environment variable for npud configuration and debug
+%define extra_option ${extra_option} -DENVVAR_NPUD_CONFIG=ON
```

Comment:
Please define different option variable. (ex. `option_debug`?)
BTW, Don't we need to use logging on release build?

Reply:
I added `option_config` define. Could you see this patch? https://github.com/Samsung/ONE/commit/02f9c350b9903cc5a7edbed4dab5e9d461e609a6

```
# Set option for configuration
%define option_config %{nil}
%if %{config_support} == 1
%if %{npud_build} == 1
# ENVVAR_NPUD_CONFIG: Use environment variable for npud configuration and debug
%define option_config -DENVVAR_NPUD_CONFIG=ON
%endif # npud_build
%endif # config_support
```